src/mpyl/utilities/repo/__init__.py

```diff
@@ -271,8 +274,9 @@ def checkout_branch(self, branch_name: str):
         self._repo.git.switch(branch_name)
 
     def does_branch_exist(self, branch_name: str, remote=False) -> bool:
+        self._repo.remote().fetch()
         found_branches = [
-            branch.strip(" ")
+            branch.strip(" ").strip("*")
```

Comment:
What is this * for?

Reply:
To remove the `*` that is placed before the current active branch when you do `git branch --list`